app.rb

```diff
@@ -47,7 +65,7 @@ def guesses
   post "/play" do
     rules = HangmanRules.new(secret_word, guesses)
     rules.guess(params['letter_chosen'], letters)
```

Comment:
Maybe the method name is a bit strange here. Because it sounds like the rules are making a guess. Maybe `rules.validate_guess` or something like that?